### PR TITLE
0.6.072 - Balance patch and spy fix

### PR DESCRIPTION
**Pull request by CDVoidwalker**

### CODE REVIEW

**Review by jrb0001 — APPROVED** [review]
Not tested but lgtm.

File: rules/defaults.yaml [hunk 1]
```diff
@@ -999,7 +1001,20 @@
 		ZOffset: 256
 		ValidStances: Ally, Neutral, Enemy
 		RequiresCondition: tiberiumheal || tiberiumheal2
-
+
```
Comment (GraionDilach) [inline rules/defaults.yaml:1004]: Superflous tabs.

File: rules/defaults.yaml [hunk 2]
```diff
@@ -999,7 +1001,20 @@
...
+		Condition: tiberiumcloak
+	Cloak:
+		RequiresCondition: cloak || cloak-bar || tiberiumcloak
+		InitialDelay: 0
+		CloakDelay: 90
+		IsPlayerPalette: True
+		CloakSound: stealth01.aud
+		UncloakSound: unstealth01.aud
+		UncloakOn: Attack, Unload, Infiltrate, Demolish, Damage
+
```
Comment (GraionDilach) [inline rules/defaults.yaml:1017]: Here too.

File: rules/misc.yaml [hunk 3]
```diff
@@ -120,9 +120,21 @@ CRATE:
 Cloud1:
 	Inherits@1: ^Clouds
 
+Cloud1Weak:
+	Inherits@1: ^CloudsWeak
+	RenderSprites:
+		Image: Cloud1
+		Scale: 0.7
```
Comment (GraionDilach) [inline rules/misc.yaml:127]: Scale should never be used and should die in a fire.

File: rules/nod-aircraft.yaml [hunk 4]
```diff
@@ -375,14 +375,14 @@ RAVEN:
...
-		Cost: 1800
+		Cost: 1500
 	Tooltip:
 		Name: Raven
 	Buildable:
 		Queue: Air.Nod
 		BuildPaletteOrder: 203
 		Prerequisites: ~nahpad, ~!t1, ~!technology
-		Description: A Jet equiped with 2 napalm rockets. Available for Industry and Propaganda Nod.\n\nGood vs: Infantry, Buildings\n\nSpecial:\n- Tiberium Weapons Research (Industry)\n- Blue Hell Inferno (Propaganda)
+		Description: A Jet equiped with a napalm rockets. Available for Industry and Propaganda Nod.\n\nGood vs: Infantry, Buildings\n\nSpecial:\n- Tiberium Weapons Research (Industry)\n- Blue Hell Inferno (Propaganda)
```
Comment (GraionDilach) [inline rules/nod-aircraft.yaml:385]: a napalm rockets.

**Pull request merged**